lib/denylist/http.go

```diff
@@ -102,12 +102,17 @@ func createDenylistEntry(response http.ResponseWriter, request *http.Request, de
 	denylist.Store(id, true)
 	log.Log.Infow("Created denylist entry", "id", id)
 	metricFilterEnabled.WithLabelValues(id).Set(1)
+	err := syncer.StoreDenylistEntry(denylist, id)
```

Comment:
I think there's an important caveat here that from the time florist sends the first request until it successfully gets a 200 from every replica, we're in an indeterminate state for that customer. I think that's ok, because if we don't get a 200 from every replica, that deploy fails anyway, and so that site is itself in an indeterminate state until we try to run the deploy again. But we should be sure to clearly document this.

Reply:
Yeah, with the caveat that the operations are all idempotent. That means that florist/oshi can assert the correct state before proceeding, and itll recover assuming everything 200s.